include/nbl/video/alloc/StreamingGPUBufferAllocator.h

```diff
@@ -14,18 +14,23 @@ namespace video
 
 class StreamingGPUBufferAllocator : protected SimpleGPUBufferAllocator
 {
+    private:
+        void* mapWrapper(IDriverMemoryAllocation* mem, IDriverMemoryAllocation::E_MAPPING_CPU_ACCESS_FLAG access, const IDriverMemoryAllocation::MemoryRange& range) noexcept;
+        void unmapWrapper(IDriverMemoryAllocation* mem) noexcept;
```

Comment:
you have to make a `.cpp` file for the wrapper's definitions anyway, so dont make wrappers and just move the methods that require them to the said `.cpp`